docs/reference/reranking/index.asciidoc

```diff
@@ -0,0 +1,71 @@
+[[re-ranking-overview]]
+= Re-ranking
```

Comment:
I'm noticing some inconsistency - sometimes we use `Reranking` and other times we use `Re-ranking`. I slightly prefer `reranking` but am fine with either, though we should do a pass for consistency.

Reply:
realized that in course of this PR and tried to align on hyphenated version, will need to update the semantic re-reranking page too

Reply:
FWIW I also prefer `reranking` but the hyphenated version seems more au courant :)